src/applet/abrt-applet-application.c

```diff
@@ -302,7 +302,8 @@ new_dir_exists (GList **new_dirs)
     cachedir = g_get_user_cache_dir ();
     dirlist_name = g_build_filename (cachedir, "abrt", NULL);
 
-    g_mkdir_with_parents (dirlist_name, 0777);
+    if (g_mkdir_with_parents (dirlist_name, 0777) != 0)
```

Comment:
Off-topic (because I see you're just being consistent with the rest of the file) but is there a reason to put a space between the function name and the opening parenthesis?

Reply:
It's only a matter of style. GLib (and GNOME/GTK?) use this convention and I think Ernestas wrote this code under that influence. The inconsistency with the rest of abrt makes me a wee bit uneasy, but I've learned to cope with it.